databricks/koalas/missing/frame.py

```diff
@@ -87,29 +87,32 @@ class _MissingPandasLikeDataFrame(object):
     to_sql = unsupported_function('to_sql')
     to_stata = unsupported_function('to_stata')
     to_timestamp = unsupported_function('to_timestamp')
+    to_markdown = unsupported_function('to_markdown')
     truncate = unsupported_function('truncate')
     tshift = unsupported_function('tshift')
     tz_convert = unsupported_function('tz_convert')
     tz_localize = unsupported_function('tz_localize')
     unstack = unsupported_function('unstack')
 
     # Deprecated functions
```

Comment:
I don't think we should necessarily classify the APIs deprecated before/after 1.0.0. You could just merge both.

Reply:
maybe it makes easy to move to pandas>=1.0.0 with dropping all the lower versions of pandas in the future??

it's not just separated into based on before and after, but whether it has been removed in pandas>=1.0.0 or not.

Reply:
We should always keep this file consistent to 1.0.0. You can remove here the methods if that's removed in the latest pandas.

Reply:
hmm. i have removed them but restore again with inspiring by https://github.com/databricks/koalas/issues/1194#issuecomment-583063664

especially.

> removed elements
> * If we already implemented, I'd prefer to keep it and show the deprecation message.
> * If it's in the missing list, I'd prefer to keep it but move it to deprecation list.


okay, but now i think the both are somewhat make sense, so let's make discussion together with @ueshin in more detail at follow-up pr will be created.